.github/workflows/publish.yml

```
@@ -38,7 +38,7 @@ jobs:
 
     - name: Publish distribution 📦 to PyPI
       if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@main
```

They actually updated example upstream.
```suggestion
      uses: pypa/gh-action-pypi-publish@release/v1
```